Source/WebCore/dom/SimpleRange.h

```diff
@@ -42,6 +83,10 @@ struct SimpleRange {
 
     bool collapsed() const { return start == end; }
 
+    operator SimpleRangeWeak() const 
+    {
+        return { BoundaryPointWeak(start.container.get(), start.offset), BoundaryPointWeak(end.container.get(), end.offset) };
+    }    
```

Comment:
This operator we actually need. It's used on line 552 of EventHandler.cpp to perform an implicit conversion from SimpleRange to SimpleRangeWeak.

Reply:
I think we should have an explicit conversion function instead. Something like `makeWeakSimpleRange`.

Reply:
Done.